auth/realm/base/src/main/java/org/wildfly/security/auth/realm/AggregateSecurityRealm.java

```diff
@@ -79,18 +79,15 @@ public AggregateSecurityRealm(final SecurityRealm authenticationRealm, Function<
     public RealmIdentity getRealmIdentity(final Evidence evidence) throws RealmUnavailableException {
         boolean ok = false;
         final RealmIdentity authenticationIdentity = authenticationRealm.getRealmIdentity(evidence);
-        if (authenticationIdentity.exists()) {
-            log.tracef("Authentication identity for principal [%s] found.", evidence.getDecodedPrincipal());
-        }
+        log.tracef("Authentication identity for principal [%s] found.", evidence.getDecodedPrincipal());
```

Comment:
I think we may need to think about the message a bit more as it is now subtly different, we can confirm we obtained a RealmIdentity from the authenticationRealm but not sure we can fully declare it "found" as we no longer have a "not found" flow.

Reply:
Hi @darranl, thank you for the review, I have updated the messages.